lib/assets/javascripts/autocomplete-rails-uncompressed.js

```diff
@@ -35,6 +35,10 @@
     var _e = e;
     this.init(_e);
   };
+  jQuery.railsAutocomplete.options = {
+    showNoMatches: 'true',
```

Comment:
@swparkaust Is there are a reason you are using a string here rather than a bool?